### ISSUE TITLE
Search Facet + Map, missing some markers?

### ISSUE THREAD

**Issue by 61pixels** [FacetWP/facetwp-map#34]
Just noticed something strange. I just added a search facet (WP default option) to my test map page. When I search, I get the correct results in the listing output (let's say 13 results in this case), but there are only 11 markers on the map. Wondering how that could happen?

I also have 3 standard facets for taxonomies with the map, and all those seem to return the correct markers/listings. It's only when I type in a search that the markers don't seem to 100% match up with the results.

**Comment by mgibbs189** [comment]
@61pixels can you confirm that all 13 results are properly geocoded (i.e. they have a latitude / longitude)?

**Comment by 61pixels** [comment]
Yes, they all are. I should mention that they are using the ACF field option, so it could be related to that. It's definitely weird. Without the search facet, and just 3 dropdown select facets the markers are there and appear fine. 

It's only when adding the search facet that some don't appear. In the case I have now, if all 3 select facets are left at the default (any), and I search for "park", 13 correct results show in the listings below the map, but only 11 markers show up. Ironically, after searching and getting the missing markers, if I then change a facet to a category that will return true for the marker, the marker appears. 

Anything else you'd like me to try?

**Comment by 61pixels** [comment]
Go ahead and call this closed, after your reply I tried a ton of different scenarios and ideas to narrow it down and came to the conclusion that nothing is wrong... Basically it turns out 3 of the locations are basically right on top of each other, something when even after zooming in previously I couldn't see them. I went ahead and turned on map clustering and the numbers all add up now, so they are there, you just have to zoom into the closest zoom to see them. Sorry if you wasted any time on this one.
